ambry-tools/src/main/java/com.github.ambry/tools/admin/ServerAdminTool.java

```diff
@@ -633,6 +678,29 @@ public ServerErrorCode controlReplication(DataNodeId dataNodeId, PartitionId par
     return adminResponse.getError();
   }
 
+  /**
+   * Sends a {@link BlobStoreControlAdminRequest} to start or stop a partition from {@code partitionIdStr}
```

Comment:
there is no `partitionIdStr`. Did you mean `partitionId`?

`Sends a {@link BlobStoreControlAdminRequest} to start or stop store associated with {@code partitionId} on {@code dataNodeId}`